signingscript/requirements/base.py38.txt

```diff
@@ -64,9 +103,9 @@ asn1crypto==1.4.0 \
     --hash=sha256:4bcdf33c861c7d40bdcd74d8e4dd7661aac320fcdf40b9a3f95b4ee12fde2fa8 \
     --hash=sha256:f4f6e119474e58e04a2b1af817eb585b4fd72bdd89b998624712b5c99be7641c
     # via mar
-async-timeout==3.0.1 \
-    --hash=sha256:0c3c816a028d47f659d6ff5c745cb2acf1f966da1fe5c19c77a70282b25f4c5f \
-    --hash=sha256:4291ca197d287d274d0b6cb5d6f8f8f82d434ed288f962539ff18cc9012f9ea3
+async-timeout==4.0.1 \
```

Comment:
Major version bump! I don't see anything in https://github.com/aio-libs/async-timeout/blob/master/CHANGES.rst that makes me concerned. Something to look at if we see bustage on rollout, though.